pulp_ansible/tests/functional/conftest.py

```diff
@@ -225,11 +226,21 @@ def _build_and_upload_collection(ansible_repo=None, **kwargs):
 
 
 @pytest.fixture
-def create_ansible_cfg(server, user):
-    return (
-        "[pulp]\nserver_list = pulp_ansible\n\n"
-        "[pulp.pulp_ansible]\n"
-        "server = {}\n"
-        "username = {}\n"
-        "password = {}"
-    ).format(server, user.username, user.password)
+def create_ansible_cfg():
```

Comment:
At this point, i think you can just make this fixture request the tmp_path fixture.
And let the factory return the path to the created file.
Remember, it is a pathlib.Path object. See this link for inspiration:
https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html#the-tmp-path-fixture

Reply:
Neat!  I would concur except that I allowed for the passed in path so this could be used in conjunction with setting the `cwd` of an `ansible-galaxy` command in a directory where it would also pick up this ansible config allowing for authentication.  I think the passed in path is preferable to string parsing in that use case?

Reply:
Do we need the directory to preexist? If not, then we could just call the the fixture `ansible_dir_factory` and return the path of the tempdir with the configuration.